helm-chart/splunk-operator/crds/enterprise.splunk.com_clustermanagers.yaml

```diff
@@ -4041,10 +4148,4 @@ spec:
     served: true
     storage: true
     subresources:
-      status: {}
-status:
-  acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: null
-  storedVersions: null
```

Comment:
is this expected?

Reply:
yes we do not have to store status here because everything is default values